### PR TITLE
Email feature

### PR DESCRIPTION
**Pull request by herwindo**
What does this PR do?
---------------------
Adds email feature
- define cc, bcc, to
- define message body like text, html

Adds local dev execution for email

How can I test this?
--------------------
1. Input credentials in scripts/local-email.js
2. Execute script by running `npm run dev:email`

Notes
-----
Do take note that you define either html or text. If you define both then email will take precedence

Input for send email are as follows:
destination: {
    to: [< email address >],
    cc,: [< email address >],
    bcc: [< email address >]
}
from: < source email >,
subject: < subject >,
html: < email template >
text: < text body >

### CODE REVIEW

File: src/lib/email.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        ToAddresses: destination.to,
+        CcAddresses: destination.cc,
+        BccAddresses: destination.bcc
+      },
+      Message: {
+        Subject: {
+          Data: subject
+        },
+        Body: {
+          ...(html ? {
```
Comment (ChanJit) [inline src/lib/email.js]: what you try to spray here?

File: scripts/local-email.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import {init, sendEmail} from '../src/lib/email';
```
Comment (ChanJit) [inline scripts/local-email.js]: dont think this need to check it, because later on will have unit test.

File: package.json [hunk 3]
```diff
@@ -5,7 +5,8 @@
   "description": "",
   "scripts": {
     "build": "npm install && gulp",
-    "test": "gulp test"
+    "test": "gulp test",
+    "dev:email": "babel-node scripts/local-email.js"
```
Comment (ChanJit) [inline package.json]: hmmm.. dont this is needed

File: src/lib/email.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import SES from 'aws-sdk/clients/ses';
+import logger from '../utilities/logger';
+
+let ses;
+
+export function init(option) {
+  const { accessKeyId, secretAccesskey, region } = option;
+
+  ses = new SES({
+    apiVersion: '2010-12-01',
```
Comment (ChanJit) [inline src/lib/email.js]: should we define our own apiVersion?

Comment (herwindo) [reply src/lib/email.js]: ok sure

File: src/lib/email.js (hunk 1 above)
Comment (herwindo) [reply src/lib/email.js]: this just deals the condition if the dev wants html or text

File: package.json (hunk 3 above)
Comment (herwindo) [reply package.json]: ok agreed

**Pull request merged**